### PR TITLE
Remove dcase

### PR DESCRIPTION
**Pull request by turian**
We are now just calling this dataset "office events", because we have modified it from the original dcase task so much.

### CODE REVIEW

File: hearpreprocess/sampler.py [hunk 1]
```diff
@@ -61,25 +60,18 @@
...
         "audio_sample_size": 4,
-        # Put two files from the dev and train split so that those splits are
-        # made
-        # dev_1_ebr_6_nec_2_poly_0.wav -> 1 train file in valid split
-        # dev_1_ebr_6_nec_3_poly_0.wav -> 1 valid file in valid split
-        "necessary_keys": [
-            "dev_1_ebr_6_nec_2_poly_0.wav",
-            "dev_1_ebr_6_nec_3_poly_0.wav",
-        ],
+        "necessary_keys": [],
```
Comment (khumairraj) [inline hearpreprocess/sampler.py:66]: Nice. The global Testing and validation percentage will handle this.

File: hearpreprocess/pipeline.py [hunk 2]
```diff
@@ -390,98 +382,103 @@ def run(self):
...
-            # Select the split to subsample
-            .loc[lambda df: df["split"] == self.split]
-        )
-        return metadata
-
     def run(self):
-        metadata = self.get_metadata()
-        num_files = len(metadata)
+        split_metadata = self.metadata[self.metadata["split"] == self.split]
+        relpaths = split_metadata["relpath"].unique()
```
Comment (khumairraj) [inline hearpreprocess/pipeline.py:423]: Maybe we can use the slug to do this sampling, so it is unique independent of the base directory. Also, the subsampling is dependent on the set of audio in the data. If we want this behaviour it is fine, but we can always use the method used in the splitting(train, test, val). i.e. by hashing followed by the division by 100 and selection.

**Review by khumairraj — APPROVED** [review]
The pipeline looks awesome! Abstractions, cleanup, going back to using simple and clear sorting to split data points and everything else look great.

File: hearpreprocess/pipeline.py (hunk 2 above)
Comment (turian) [reply hearpreprocess/pipeline.py:423]: https://github.com/neuralaudio/hear-preprocess/pull/13

**Pull request merged**